### PR TITLE
Deal with backbone with only un-oriented sequences

### PR DESCRIPTION
**Pull request by jowong4**
There are edges cases where a backbone consist of only un-oriented sequences. This PR contains code to deal with these exceptions, by taking the longest sequence within the backbone and assigning it a positive orientation

### CODE REVIEW

File: physlr/physlr.py [hunk 1]
```diff
@@ -2195,14 +2195,23 @@ def physlr_path_to_fasta(self):
...
+            for idx, name in enumerate(path):
+                if name[-1] != ".":
+                    all_unoriented = False
+                    break
+                if len(name) > len(path[longest_seq]):
+                    longest_seq = idx
+            if all_unoriented:
+                path[longest_seq] = path[longest_seq][:-1] + "+"
+
             seq = gaps.join(Physlr.get_oriented_sequence(seqs, name)
```
Comment (lcoombe) [inline physlr/physlr.py]: Wouldn't we get the same sequences outputted if we check if all the sequences are unoriented, and just move on to the next path in those cases? So just don't place anything here if all are unoriented, then the code below will take care of placing that sequence. 
Just wondering if that's cleaner.

Comment (jowong4) [reply physlr/physlr.py]: Your solution is more correct. It just becomes harder for me to track down the backbone. That's why I did this. Thought I suppose in the end, it doesn't really matter.

Comment (jowong4) [reply physlr/physlr.py]: addressed

File: physlr/physlr.py [hunk 2]
```diff
@@ -2195,11 +2195,20 @@ def physlr_path_to_fasta(self):
...
+            all_unoriented = True
+            longest_seq = 0
+            for idx, name in enumerate(path):
+                if name[-1] != ".":
+                    all_unoriented = False
+                    break
+                if len(name) > len(path[longest_seq]):
+                    longest_seq = idx
+            if all_unoriented:
+                path = None
```
Comment (lcoombe) [inline physlr/physlr.py]: What about something like:
```
all_unoriented = all([name[-1] == "." for name in path])
if all_unoriented:
    continue
```
 I don't think you need the length tracking anymore

Comment (jowong4) [reply physlr/physlr.py]: That is correct. Didn't know there there was an `all` function

**Pull request merged**